src/datasets/commands/dummy_data.py

```diff
@@ -1,468 +0,0 @@
-import fnmatch
```

Comment:
The removal of deprecated code has been addressed in a separate dedicated PR:
- #6996: https://github.com/huggingface/datasets/pull/6996/commits/5d4687e4ee1dba2b14e59117492d7f75ea66800a